internal/cephfs/controllerserver.go

```diff
@@ -187,7 +187,6 @@ func (cs *ControllerServer) CreateVolume(
 	if req.GetCapacityRange() != nil {
 		volOptions.Size = util.RoundOffBytes(req.GetCapacityRange().GetRequiredBytes())
 	}
-	// TODO need to add check for 0 volume size
```

Comment:
From CSI spec

```
// This field is OPTIONAL. This allows the CO to specify the capacity
  // requirement of the volume to be provisioned. If not specified, the
  // Plugin MAY choose an implementation-defined capacity range. If
  // specified it MUST always be honored, even when creating volumes
  // from a source; which MAY force some backends to internally extend
  // the volume after creating it.
  CapacityRange capacity_range = 2;
```

are we sure that no CO is implementing the `0` size? because we have some users for kubernetes and Nomad CO

Reply:
@madhu-1 https://github.com/ceph/ceph-csi/pull/2472#issuecomment-910122520 is on Kube CO , reg#` nomad`,  I went through available articles  to an extent,  but was not able to find any particular reference on `zero` sized PVCs.  I am aware from a couple of recently reported nomad issues here in this repo that,  ceph csi is conusumed with Nomad too, eventhough we havent really paid an attention specific to `nomad` CO in many places of the code, and if we start accounting Nomad as a CO ,  Iam in an impression that, many places to revisit in ceph csi and check the different possibilities. with that in mind , do we really need to worry about `nomad` at this stage ? also, do you see any reference form Nomad where they create `0` sized PVCs?  Cc @nixpanic

Reply:
Also as an additional reference to CSI spec, see https://github.com/container-storage-interface/spec/issues/488

Reply:
I really don't know about the nomad but at the same time, I don't think we need to consider removing the TODO unless this is mentioned in the CSI spec.

Reply:
@nixpanic @madhu-1 one other thought I have here is to mention the CO we support in this project . Considering we are doing or testing only Kube , may be we have to report it in the support matrix too.  once we started to qualify `nomad` or some other `CO` we can update the matrix. Please let me know what do you think , accordingly I can update the support matrix too .

Reply:
Yes that looks good 👍

Reply:
I would not like to see 'support' for specific COs. Maybe list 'known to work COs' and list the versions we use in CI and known products? That way, users that manage to configure Hashicorp Nomad, Apache Mesos or Docker Swarm could add the versions they use.

Reply:
yeah, that make sense, let me file a PR with `known to work COs`.  :+1:

Reply:
@nixpanic @Madhu-1 added `Known to Work` COs and tested matrix. PTAL. Thanks